core/src/components/toast/test/buttons/toast.e2e.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs().forEach(({ title, config }) => {
+  test.describe(title('toast: cancel button'), () => {
+    test('should render the cancel button with part button-cancel', async ({ page }) => {
+      await page.goto('/src/components/toast/test/buttons', config);
+
+      const toast = page.locator('#closeArray');
+      await toast.click();
+
+      const buttonCancel = page.locator('.toast-button-cancel');
+      await expect(buttonCancel).toHaveAttribute('part', 'button-cancel');
```

Comment:
Can we make this a spec test? E2E tests fire up an entire browser instance which is slow and not necessary for the kind of validation we are trying to do.

Here's an example of a spec test in Ionic: https://github.com/ionic-team/ionic-framework/blob/1cf1eca00239f3e98854466116e42f9a2e7b4590/core/src/components/select/test/select.spec.tsx#L2